docs/content/docs/concepts/signals.md

```diff
@@ -6,6 +6,92 @@ discussion: 5469
 aliases:
   - "/docs/advanced/signals/"
   - "/docs/deep/signals/"
+draft: true
 ---
 
-Signals are similar to ops, but are not stored; they're useful to implement features like user presence.
+When using DDSes, data is sequenced and stored within the Fluid container to achieve synchronized shared state. For scenarios that involve shared persisted data, DDSes provide an effective way to communicate data so that it is retained in the container. However, there are many scenarios where we need to data communicate that is short-lived, in which the ordering and storage of said information would be wasteful and unnecessary.
```

Comment:
```suggestion
When using DDSes, data is sequenced and stored within the Fluid container to achieve synchronized shared state. For scenarios that involve shared persisted data, DDSes provide an effective way to communicate data so that it is retained in the container. However, there are many scenarios where we need to data communicate that is short-lived, in which the ordering and storage of said information would be wasteful and unnecessary.
```